components/edit_post_modal/edit_post_modal.test.tsx

```diff
@@ -109,8 +126,8 @@ describe('components/EditPostModal', () => {
             title: 'test',
         };
 
-        var wrapper = shallowWithIntl(createEditPost({actions, editingPost}));
-        var instance = wrapper.instance();
+        const wrapper: ShallowWrapper<any, any, any> = shallowWithIntl(createEditPost({...defaultProps, actions, editingPost}));
```

Comment:
Same here, any suggestions on how to type this?

Reply:
Unfortunately with `shallowWithIntl`, the typing isn't correct with respect to what component it returns in the `ShallowWrapper`.

What I would do here if you need to access methods of the class component is to export the components type as non-default, and cast the instance as the component type (you might need to cast to `unknown` first).

In this particular test it looks like no specific instance method needs to be called, other than default React methods so you might be able to avoid any casting.

Reply:
Ah, that makes sense. I removed all the `ShallowWrapper<any, any, any> except in this `testComponentForMarkdownHotkeys` call. I'm not really sure what that should be. I also had to add some other `any` to get it to compile again and am not sure if that is ok either.